src/drivers/esp8266_drv.c

```diff
@@ -672,6 +683,168 @@ static void check_wifi_client()
 }
 
 
+/* *** Methods that handle the Wifi AP and its actions *** */
+
+/**
+ * Callback that gets invoked when the get_ap_info command completes.
+ */
+static void get_ap_info_cb(const bool status,
+                           const struct esp8266_ap_info* info)
+{
+        cmd_completed();
+        cmd_set_check(CHECK_WIFI_AP);
+        esp8266_state.ap.info_timestamp = getUptime();
+
+        const size_t ap_info_size = sizeof(esp8266_state.ap.info);
+        if (!status) {
+                pr_warning(LOG_PFX "Failed to read AP info\r\n");
+                memset(&esp8266_state.ap.info, 0, ap_info_size);
+                return;
+        }
+
+        memcpy(&esp8266_state.ap.info, info, ap_info_size);
+
+        /* Print out our AP Info here */
+        pr_info(LOG_PFX "AP info:\r\n");
+        pr_info_str_msg("\t SSID      : ", info->ssid);
+        pr_info_str_msg("\t Password  : ", info->password);
```

Comment:
Thought about that.  We will probably delete these before we ship.  I wanted them for debugging.  Tracked by #606